utils.py

```diff
@@ -253,6 +253,22 @@ def write_vcf(lines, vcf):
     return vcf
 
 
+def write_filtered_vcf(vcf, chrm, out_vcf):
+
+    content = []
+
+    with versatile_open(vcf, "r") as vcf_handle:
+
+        for line in vcf_handle.readlines():
```

Comment:
Is there a reason for loading the VCF in memory? Why not simply `for line in vcf_handle:`?

Reply:
Good spot, I'll change this